readthedocs/rtd_tests/tests/test_config_integration.py

```diff
@@ -729,43 +729,6 @@ def test_sphinx_configuration(
         append_conf.assert_called_once()
         move.assert_called_once()
 
-    @patch('readthedocs.doc_builder.backends.sphinx.BaseSphinx.move')
-    @patch('readthedocs.doc_builder.backends.sphinx.BaseSphinx.append_conf')
-    @patch('readthedocs.doc_builder.backends.sphinx.BaseSphinx.run')
-    def test_sphinx_fail_on_warning(
```

Comment:
Not sure how this landed here... Anyway this test was added in  https://github.com/rtfd/readthedocs.org/pull/4456/files#diff-5a2cc4147895068b33fe5bf89e4e6e08R685